handsontable/src/focusManager.js

```diff
@@ -0,0 +1,217 @@
+import { warn } from './helpers/console';
+
+/**
+ * Possible focus modes.
+ * - CELL - The browser's focus stays on the lastly selected cell element.
+ * - MIXED - The browser's focus switches from the lastly selected cell element to the currently active editor's
+ * `TEXTAREA` element after a delay defined in the manager.
+ *
+ * @type {{CELL: string, MIXED: string}}
+ */
+const FOCUS_MODES = Object.freeze({
+  CELL: 'cell',
+  MIXED: 'mixed',
+});
+
+/**
+ * Manages the browser's focus in the table.
+ */
+export class FocusManager {
+  /**
+   * The Handsontable instance.
+   *
+   * @private
+   */
+  #hot;
```

Comment:
```suggestion
  /**
   * The Handsontable instance.
   */
  #hot;
```
If the field is declared as private then it won't show on Docs. The additional JSDoc marker is redundant.